### PR TITLE
Add enter shortcut for class creation dialog.

### PR DESCRIPTION
**Pull request by Jonas-Sander**
When entering the school class name inside the school class creation dialog and pressing "Enter" the school class will now be created.

Before this change the TextField just lost focus after pressing "Enter".

### CODE REVIEW

File: app/lib/groups/src/pages/school_class/school_class_create.dart [hunk 1]
```diff
@@ -6,64 +6,81 @@
...
       content: SingleChildScrollView(
         child: Column(
           crossAxisAlignment: CrossAxisAlignment.start,
           children: <Widget>[
             const SizedBox(height: 8),
-            _NameField(),
+            _NameField(
+              value: className,
+              onChanged: (newName) => setState(() => className = newName),
+              onSubmitted: (className) => _createClass(context, className),
```
Comment (Jonas-Sander) [inline app/lib/groups/src/pages/school_class/school_class_create.dart:74]: This is the change for the "Enter" behavior

**Comment by Jonas-Sander** [comment]
Otherwise I did a little refactoring. Instead of using an extra "Bloc" (that isn't a Bloc) we just use the StatefulWidget directly to save the class name.  
The Bloc was used elsewhere though so I moved the Bloc to the other location it was used.

File: app/lib/groups/src/pages/school_class/school_class_create.dart (hunk 1 above)
Comment (Jonas-Sander) [reply app/lib/groups/src/pages/school_class/school_class_create.dart:74]: (`onSubmitted`)

**Review by nilsreichardt — APPROVED** [review]
LGTM

But let's set up a web preview for every pull request, so we are able to quickly test these changes. I already started with #116.

**Pull request merged**